### ISSUE TITLE
[Effect Suggestion] Fake effect

### ISSUE THREAD

**Issue by MoDerPo** [gta-chaos-mod/ChaosModV#552]
**Name of effect**
Fake effect

**Effect Description**
displays a random effect that doesn't actually happen and when it would be over it says that it was fake

**Comment by TheRealAyCe** [comment]
Should probably say that after some seconds already. And then switch the name to "Bamboozled!".

**Comment by kolyaventuri** [comment]
We already have a handful of fake effects (fake crash, fake teleport, etc.)... Do you just mean it would pick _any_ effect and display it as fake? That might work well for a handful of effects, however some stuff like Fullbright or "Snowy Weather" it would be super obvious that it's fake...

**Comment by TheRealAyCe** [comment]
...which seems fine. Not every effect is dramatic or interesting.

"Nothing"...? owww actually the Fake version. Bamboozled again.

**Comment by SeanMirrsen** [comment]
This one should be cheap once the fake names for fake effects PR is merged. Just pick a random effect name to display, keep it there for a few seconds, then change to 'Bamboozled!'.

**Comment by MoDerPo** [comment]
> ...which seems fine. Not every effect is dramatic or interesting.
> 
> "Nothing"...? owww actually the Fake version. Bamboozled again.

hahaha

**Comment by MoDerPo** [comment]
> This one should be cheap once the fake names for fake effects PR is merged. Just pick a random effect name to display, keep it there for a few seconds, then change to 'Bamboozled!'.

I don't get what your saying... and what you mean by PR

**Comment by kolyaventuri** [comment]
> > This one should be cheap once the fake names for fake effects PR is merged. Just pick a random effect name to display, keep it there for a few seconds, then change to 'Bamboozled!'.
> 
> I don't get what your saying... and what you mean by PR

We have a PR (Pull Request, a set of proposed changes to the code that must be approved before they are approved and made permanent) in progress that will allow us to make certain effects have "Fake" names. Such as "Fake Death" showing "Suicide" initially, and switching to "Fake Death" once it actually reveals that

**Comment by MoDerPo** [comment]
oh now I get it... But that's not what I meant. I meant that it displays for example "Invincible" but nothing actually happens. Then you relie on it and after a bit it says "Bamboozled!". Of course it could also be something obvious but then you'd just be confused.

**Comment by SeanMirrsen** [comment]
I meant that once that PR lands (i.e., gets implemented), you could simply define an empty effect with a selection of fake names. It'd be cheap to do it at that point, without having to create special functionality for this one.

I'm not against the effect, it seems a nice little fakeout.

**Comment by MoDerPo** [comment]
yes but we're not talking about the same effect

**Comment by MoDerPo** [comment]
I mean it says it's doing an effect but it doesn't.

**Comment by MoDerPo** [comment]
and I think what you're talking about is a decoy name for the Fake Teleport, Fake crash and Fake Death effects

**Comment by SeanMirrsen** [comment]
...

The pull request will only add those fake names to those specific effects, yes.
However the functionality to add fake names to effects, right now, does not exist.
Once the PR is merged, the functionality will be there.
Therefore allowing this effect to be made very easily.

Make sense now?

**Comment by MoDerPo** [comment]
Yes it makes sense but I don't get why we want that functionality... Do you mean that there should be a list of all effect names that my meta effekt would acces?

**Comment by SeanMirrsen** [comment]
What your effect does, is the same as the already-existing effect, Nothing. If it activates, it must display either its true name, or nothing.

Without the ability to display fake names, it can't display the name of another effect before switching to 'Bamboozled!' or whichever it will be. You want that functionality in order for your effect to actually work the way you want it to.

**Comment by MoDerPo** [comment]
oh ok now I get it yeah

**Comment by Da-Best** [comment]
I think just removing "nothing" and replacing with this would be better then adding a new effect in game that is basically the same as "nothing"

**Comment by JustTunerYT** [comment]
> I think just removing "nothing" and replacing with this would be better then adding a new effect in game that is basically the same as "nothing"

I agree, because this effect would do just the same as "Nothing".
_Unless "nothing" will be replaced with SOMETHING_

**Comment by MoDerPo** [comment]
well it is effectively nothing but it makes the player paranoid that the effect is about to occur or they may even think that something is caused by the effect, even tho it's just normal gameplay. You would also never know if any effect is really happening because it could just be fake.
